gobcore/model/gobmodel.json

```diff
@@ -44,11 +44,6 @@
         "description": "Een adres in de nabijheid van de meetbout",
         "ref": "nummeraanduiding"
       },
-      "ligt_in": {
-        "type": "GOB.String",
-        "description": "Het bouwblok waarbinnen de meetbout ligt",
-        "ref": "bouwblok"
-      },
       "ligt_in": {
```

Comment:
My mistake, ligt_in_bouwblok and ligt_in_stadsdeel are two distinct fields. I was too quick with getting the ref out of the name and put into an attribute. So instead of removing the "duplicated" ligt_in, it should be well named. In order to stay consistent the other ref field "nabij" could also be given a more declarative name like nakij_adres or something